pkgs/games/frogatto/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, buildEnv, stdenv, callPackage, makeWrapper, makeDesktopItem,
+}:
+
+let
+  description = "Action-adventure game, starring a certain quixotic frog";
+  engine = callPackage ./engine.nix { };
+  data = callPackage ./data.nix { };
+  desktopItem = makeDesktopItem {
+    name = "frogatto";
+    exec = "frogatto";
+    startupNotify = "true";
+    icon = "${data}/share/frogatto/modules/frogatto/images/os/frogatto-icon.png";
+    comment = description;
+    desktopName = "Frogatto";
+    genericName = "frogatto";
+    categories = "Application;Game;ArcadeGame;";
+  };
+in buildEnv rec {
+  name = "frogatto-${version}";
+  version = "unstable-2018-12-18";
+
+  buildInputs = [ makeWrapper ];
+  paths = [ engine data desktopItem ];
```

Comment:
I don't think `desktopItem` should be here if we're copying it in `postBuild`.

Reply:
Now just linking it in `postBuild`.